gandalf.js

```diff
@@ -5,41 +5,45 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.